Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -1569,6 +1580,73 @@ - (IBAction)showImageChoices:(id)sender {
     [self beginTaskWithIdentifier:ImageChoicesTaskIdentifier];
 }
 
+# pragma mark - Image Capture
+- (id<ORKTask>)makeImageCaptureTask {
+    NSMutableArray *steps = [NSMutableArray new];
+    
+    {
+        ORKInstructionStep *step = [[ORKInstructionStep alloc] initWithIdentifier:@"begin"];
+        step.title = @"Hands";
+        step.image = [UIImage imageNamed:@"hands_solid"];
```

Comment:
So then, you are saying that if an image was given to the `ORKTintedImageView` that had a `renderingMode` of `UIImageRenderingModeAutomatic`, then whether it is shown as a template image or not should be governed by the `shouldApplyTint` property of `ORKTintedImageView`.  If the `renderingMode` of the image given to the `ORKTintedImageView` is either `UIImageRenderingModeAlwaysTemplate` or `UIImageRenderingModeAlwaysOriginal`, then you do not want `ORKTintedImageView` to change it, no matter what the `shouldApplyTint` property is set to.  Is that right?